platform/platform.go

```diff
@@ -136,6 +136,32 @@ func (t *TestCluster) DropFile(localPath string) error {
 	return nil
 }
 
+func (t *TestCluster) err(e error) {
+	panic(e)
+}
+
+func (t *TestCluster) Error(e error) {
+	t.err(e)
+}
+
+func (t *TestCluster) Errorf(format string, args ...interface{}) {
+	t.err(fmt.Errorf(format, args...))
+}
+func (t *TestCluster) skip(why string) {
+	panic(util.Skip(why))
+}
+
+func (t *TestCluster) Skip(args ...interface{}) {
+	t.skip(fmt.Sprint(args...))
+}
+
+func (t *TestCluster) SkipNow() {
```

Comment:
testing may have this one but lets not bother, requiring skips to have a message.